app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapControls.jsx

```diff
@@ -369,14 +394,14 @@ export default class SamplesHeatmapControls extends React.Component {
       <div className={cs.menu}>
         <Divider />
         <div className={`${cs.filterRow} row`}>
-          <div className="col s3">{this.renderTaxonLevelSelect()}</div>
+          <div className="col s2">{this.renderTaxonLevelSelect()}</div>
```

Comment:
Is there a reason this was changed to `s2`? It leaves some empty space on the right of the filters.

Reply:
No good reason. I'll change it back.